frontend/i-league-trophy-explorer/style.css

```
@@ -0,0 +1,521 @@
+    content: counter(ref) ".";
+    position: absolute;
+    left: 1rem;
+    top: 1rem;
+    color: var(--saffron, #FF9933);
+    font-weight: 700;
+}
+
+.references-list .reference-item a {
+    color: var(--saffron, #FF9933);
+    text-decoration: none;
+    word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)